.circleci/config.yml

```diff
@@ -1,28 +1,27 @@
-version: 2
+version: 2.1
+orbs:
+    gh: circleci/github-cli@1.0.3
 jobs:
     build:
         docker:
             - image: circleci/node:12.6-browsers
         steps:
             - checkout
-            - restore_cache:
```

Comment:
no more caching dependencies?

Reply:
Yep this was dropped accidently and has been added back in.